packages/dev/parcel-transformer-mdx-docs/MDXTransformer.js

```diff
@@ -59,8 +59,10 @@ module.exports = new Transformer({
 
             // TODO: Parsing code with regex is bad. Replace with babel transform or something.
             let code = node.value;
-            code = code.replace(/import ((?:.|\n)*?) from (['"].*?['"]);?/g, (m, _, s) => {
-              if (s.slice(1, -1) !== 'your-component-library') {
+            code = code.replace(/import ((?:.|\n)*?) from (['"].*?['"]);?/g, (m, x, s) => {
+              if (meta === 'example' && s[1] === '.') {
+                exampleCode.push(`import ${x} from "extract:${s.slice(1, -1)}.mdx";`);
```

Comment:
Basically if you import a relative path (starting with "./") from an MDX file, it will import the Reusable Wrappers section of the referenced file. That allows components to be reused between pages.